### PR TITLE
pacman*: add Turkish translations

### PR DESCRIPTION
**Pull request by Yutyo**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.tr/linux/pacman-mirrors.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# pacman-mirrors
+
+> Manjaro Linux için pacman aynalistesi oluşturucu.
+> pacman-mirrors'ın çalıştırıldığı her vakit, E`sudo pacman -Syyu` komutu ile veritabanının senkronize edilmesi ve sistemin güncellenmesi gerekir.
+> Daha fazla bilgi için: <https://wiki.manjaro.org/index.php?title=Pacman-mirrors>.
```
Comment (navarroaxel) [inline pages.tr/linux/pacman-mirrors.md:5]: is `Daha fazla bilgi` a more exact translation from `More information`?
```suggestion
> Daha fazla bilgi: <https://wiki.manjaro.org/index.php?title=Pacman-mirrors>.
```

File: pages.tr/linux/pacman-query.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- İndirilmiş bir pakete dair bilgiyi görüntüle:
+
+`pacman --query --info {{paket_ismi}}`
+
+- Bir paketin içerdiği dosyaları sırala:
+
+`pacman --query --list {{paket_ismi}}`
+
+- Yetim (başka bir pakete bağlılık olarak indirilmiş ancak herhangi bir paket tarafından
+  gerektirilmeyen) paketleri sırala:
```
Comment (navarroaxel) [inline pages.tr/linux/pacman-query.md:27]: Just leave the example description in one line:
```suggestion
- Yetim (başka bir pakete bağlılık olarak indirilmiş ancak herhangi bir paket tarafından gerektirilmeyen) paketleri sırala:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.tr/linux/pacman-mirrors.md (hunk 1 above)
Comment (Yutyo) [reply pages.tr/linux/pacman-mirrors.md:5]: `Daha fazla bilgi için` makes more sense in Turkish language.

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]
Cool!!!!!!!!!!!!!!!

**Pull request merged**